instagram-page-react/src/molecules/header/atoms/headerActions/HeaderActions.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+// Lodash
+import _map from 'lodash/map';
+
+// Styles
+import styles from "./headerActions.module.css";
+
+// constants
+import {EMPTY_OBJECT} from "../../../../app.constants";
+import {headerActionsConstants} from "./headerActions.constants";
+
+const renderHeaderAction = (headerAction = EMPTY_OBJECT) => {
+    const {
+        linkTo,
+        ID,
+        actionText
+    } = headerAction
+
+    return (
+        <li key={linkTo}>
+            <a
+                className={ID==="SIGNUP" ? styles.actionSignup : styles.action}
```

Comment:
Class should be config-driven, this way we don't need to have a check over `id` here